src/ResourceManager/ContainerRegistry/ChangeLog.md

```diff
@@ -0,0 +1,30 @@
+<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 1.0.0
```

Comment:
It seems odd to have change logs without saying what version it is (current doesn't say version though). Is this something new since I didn't find another module doing what you were suggesting?